src/app/Library/CrudPanel/Traits/Query.php

```diff
@@ -142,4 +142,13 @@ public function count()
     {
         return $this->query->count();
     }
+
+    public function applyCrudOrder($column_name, $column_direction = 'ASC')
+    {
+        if (! $this->driverIsMongoDb()) {
```

Comment:
I take it you didn't find a way to check if it's an SQL driver (more general)? We have to hardcode it for every noSQL driver?

Reply:
Also, in general this kind of methods are easier to read if the conditional does not test for something negative, but for something positive. There's no processing difference, just a difference in how easy it is to read.  So:

```php
if ($this->driverIsMongoDb()) {
   return 'smth';
}

return 'smth_else';
```

It basically uses the "escape early" paradigm. For these methods you'd expect the _most likely_ scenario to be at the end. And all exceptions to be first, returning early.

Reply:
The only solution I can think of to check for sql drivers is to use an helper function like we do for mongo:

```php
    private function isSqlDriver() {
        $driver = $this->getSchema()->getConnection()->getConfig()['driver'];

        if(in_array($driver, ['mysql', 'sqlsrv', 'sqlite', 'pgsql'])) {
            return true;
        }

        return false;
    }

```

If you are ok with this solution I can push the fixes in the PR.

Best,
Pedro

Reply:
Got it. Yeah, I agree with you then - that's not better in any way, and we already have `driverIsMongoDb()`. So we might as well use it 😄

Reply:
@pxpm I agree, `isSqlDriver` is safer, that way we know this wont fail on other database engines.

Reply:
Btw, we should stick with the previous naming syntax, `driverIsMongoDb`, so `driverIsSqlDb`, right?

    private function driverIsSqlDb()
    {
        $driver = $this->getSchema()->getConnection()->getConfig()['driver'];
        return in_array($driver, ['mysql', 'sqlsrv', 'sqlite', 'pgsql']);
    }